Formula/mbt.rb

```diff
@@ -0,0 +1,25 @@
+class Mbt < Formula
+  desc "Multi-Target Application (MTA) build tool for Cloud Applications"
+  homepage "https://sap.github.io/cloud-mta-build-tool"
+  url "https://github.com/SAP/cloud-mta-build-tool/archive/v1.2.10.tar.gz"
+  sha256 "46d548342e2fe3abd51d4c5d5702602b835a2cfd53eb09b86a2abf45825823a4"
+  license "Apache-2.0"
+  head "https://github.com/SAP/cloud-mta-build-tool.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["CGO_ENABLED"] = "0"
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/SAP/cloud-mta-build-tool").install buildpath.children
+    cd "src/github.com/SAP/cloud-mta-build-tool" do
+      system "go", "generate"
+      system "go", "build"
+      bin.install "cloud-mta-build-tool" => "mbt"
+    end
+  end
+
+  test do
+    assert_match(/Cloud MTA Build Tool/, shell_output("#{bin}/mbt --version"))
+  end
```

Comment:
def install
    ldflags = %W[
      -s -w
      -X main.Version=#{version}
      -X main.BuildDate=#{time.iso8601}
    ]

    system "go", "build", *std_go_args(ldflags: ldflags)
  end

  test do
    assert_match /generating the \"Makefile_\d+.mta\" file/, shell_output("#{bin}/mbt build", 1)
    assert_match(/Cloud MTA Build Tool/, shell_output("#{bin}/mbt --version"))
  end